<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 204, 234);
background: rgb(255, 253, 246);</string>
    </property>
    <property name="text">
     <string>Phương trình bậc 2</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxHeSo">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>521</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(224, 250, 255);</string>
    </property>
    <property name="title">
     <string>Nhập các hệ số của phương trình:</string>
    </property>
    <widget class="QLabel" name="labelHeSoA">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>52</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Hệ số a:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditHeSoA">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>391</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditHeSoB">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>391</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelHeSoB">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>52</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Hệ số a:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditHeSoC">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>391</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelHeSoC">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>52</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Hệ số a:</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxChucNang">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>250</y>
      <width>521</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(233, 225, 242);</string>
    </property>
    <property name="title">
     <string>Các chức năng:</string>
    </property>
    <widget class="QPushButton" name="pushButtonGiai">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(255, 231, 188);</string>
     </property>
     <property name="text">
      <string>Giải</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>images/ic_calculate.png</normaloff>images/ic_calculate.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonTiep">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(255, 231, 188);</string>
     </property>
     <property name="text">
      <string>Tiếp</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>images/ic_new.png</normaloff>images/ic_new.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonThoat">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>30</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(255, 231, 188);</string>
     </property>
     <property name="text">
      <string>Thoát</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>images/ic_exit.png</normaloff>images/ic_exit.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxKetQua">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>390</y>
      <width>521</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Kết quả:</string>
    </property>
    <widget class="QLabel" name="labelKetQua">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>471</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(255, 246, 227);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>703</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
